packages/php/src/Metrics.php

```diff
@@ -86,9 +87,13 @@ public function __construct(string $api_key, string $group_handler, array $optio
      * @param Response $response
      * @throws MetricsException
      */
-    public function track(Request $request, $response): void
+    public function track(Request $request, &$response): void
     {
-        $payload = $this->constructPayload($request, $response);
+        $log_id = Uuid::uuid4()->toString();
+        $response->headers->set('x-readme-log', $log_id);
```

Comment:
Shall we rename this to x-documentation-url to match the node package? Or is it still like this?

Reply:
Going to leave that as it is for this PR and make those changes in https://github.com/readmeio/metrics-sdks/pull/15 as it  requires a bit larger overhaul.